S/SHTns/build_tarballs.jl

```diff
@@ -3,44 +3,52 @@
 using BinaryBuilder, Pkg
 
 name = "SHTns"
-version = v"3.5.1"
+version = v"3.5.2"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://bitbucket.org/nschaeff/shtns/downloads/shtns-$(version).tar.gz", "77f8a33b94df8786d2ce9b95cbfbe548f00443625b310b8c64012b22c8a7394f")
+    ArchiveSource("https://bitbucket.org/nschaeff/shtns/downloads/shtns-$(version).tar.gz", "dc4ac08c09980e47c71d79d38696c5d1d631f86c2af1ce8aad5d21f7fd2c05b9")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/shtns/
-export CFLAGS="-fPIC"
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
-if [[ "${target}" == *-apple-* ]]; then
-    # For some reasons, `configure` insists on setting CC2=gcc, also on macOS
-    sed -i -e 's/gcc/cc/' -e 's/ -fno-tree-loop-distribute-patterns//' Makefile
-fi
```

Comment:
Was that fixed?

Reply:
Oh, that probably can be fixed with `--enable-kernel-compiler=cc`

Reply:
yes, that build was before your comment, without the flag